opensearch-observability/build.gradle

```diff
@@ -23,7 +23,6 @@ buildscript {
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)
         kotlin_version = System.getProperty("kotlin.version", "1.6.0")
         job_scheduler_version = System.getProperty("job_scheduler.version", opensearch_build)
-        jackson_version = "2.14.1"
```

Comment:
curious as to do we need to add a line like they did in sql : https://github.com/opensearch-project/sql/pull/1169/files?

Reply:
`apply plugin: 'opensearch.java'`

Reply:
I think SQL needs it as they are still using the `versions.jackson` from the OpenSearch config. This may be needed for SQL due to its direct dependency from other gradle libraries. We don't need it as we just use the version provided by the OS-core.